spec/lib/miq_expression/tag_spec.rb

```diff
@@ -2,19 +2,54 @@
 
 RSpec.describe MiqExpression::Tag do
   describe ".parse" do
-    it "with managed-field" do
-      field = "managed.location"
-      expect(described_class.parse(field)).to eq(described_class.new(nil, "/location"))
-    end
-
     it "with model.managed-in_field" do
       field = "Vm.managed-service_level"
-      expect(described_class.parse(field)).to eq(described_class.new(Vm, "/managed/service_level"))
+      expect(described_class.parse(field)).to eq(described_class.new(Vm, "/managed/service_level", []))
     end
 
     it "with model.last.managed-in_field" do
       field = "Vm.host.managed-environment"
-      expect(described_class.parse(field)).to eq(described_class.new(Vm, "/host"))
+      expect(described_class.parse(field)).to eq(described_class.new(Vm, "/managed/environment", ['host']))
```

Comment:
@imtayadeway  I agree with you so I removed `eql?`  and I used `have_attributes`